satellite/marketing/marketingweb/server.go

```diff
@@ -79,16 +114,23 @@ func NewServer(logger *zap.Logger, config Config, listener net.Listener) (*Serve
 	return s, nil
 }
 
-// ServeHTTP handles index request
-func (s *Server) ServeHTTP(w http.ResponseWriter, req *http.Request) {
+// getOffers renders the tables for free credits and referral offers to the UI
+func (s *Server) getOffers(w http.ResponseWriter, req *http.Request) {
 	if req.URL.Path != "/" {
 		s.serveNotFound(w, req)
 		return
 	}
 
-	err := s.templates.home.ExecuteTemplate(w, "base", nil)
+	offers, err := s.service.ListAllOffers(context.Background())
```

Comment:
`context.Background()` shouldn't be used: https://github.com/storj/storj/wiki/Style#contexts

Reply:
`req.Context()` ?